scripts/validate_docstrings.py

```diff
@@ -42,6 +42,7 @@
 
 
 PRIVATE_CLASSES = ['NDFrame', 'IndexOpsMixin']
+DIRECTIVES = ['.. versionadded', '.. versionchanged', '.. deprecated']
```

Comment:
afaik, all sphinx directives start with `.. ` so I think it's better to have just the names here.